src/process_utility.c

```diff
@@ -1114,6 +1114,46 @@ process_truncate(ProcessUtilityArgs *args)
 							 */
 							list_changed = true;
 					}
+					else if ((chunk = ts_chunk_get_by_relid(relid, false)) != NULL)
+					{
```

Comment:
Could need a comment here what this is about. This is about truncating a single chunk?